### ISSUE TITLE
Address insufficient contrast of white text over grey background: 0.5

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#251]
e.g., in breadcrumbs for people sections.

> The color #FFF with background color #B5B2B0 has a contrast ratio of 2.1:1, which fails for normal sized text. 

![screen shot 2018-04-24 at 3 16 55 pm](https://user-images.githubusercontent.com/263285/39211869-8dcc55f6-47d2-11e8-88fd-bb17d4b41f98.png)

Traceability: [Microassist Report 1 > WCAG Compliance > Color Contrast](https://docs.google.com/document/d/10cfUFx-o_kSGBFKCe4MtL45cyc-c-Gwaz3Dxlrp8Qgg/edit#bookmark=id.fg49504ke5mr)

WCAG 2.0: [1.4.3 Contrast (Minimum)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html): The visual presentation of texts and images of text has a contrast ratio of at least 4.5:1 for normal text and 3:1 for large text. 14 point and bold or larger, or 18 point and larger. (Level AA)

**Comment by paregorios** [comment]
In order to achieve proper contrast, make the following CSS changes:

## background of gray breadcrumbs:

```css
#portal-breadcrumbs {
    background: #ddd9d6;
    ...
```

## links, text, and separators for the breadcrumbs:

```css
color: #333;
```

Result should look like this:

![screen shot 2018-12-03 at 10 52 59 am](https://user-images.githubusercontent.com/263285/49388474-a1f80900-f6e9-11e8-8701-aec9ca79d427.png)

**Comment by alecpm** [comment]
This is live on staging (whenever it's back online)

**Comment by alecpm** [comment]
I do think this needs a better focus indicator. Perhaps adding `padding: 5px; border: 3px solid transparent;` and on hover/focus `border-color: blue;` with a reduction in right margin from 14px to 8px to account for the additional border and padding.

**Comment by alecpm** [comment]
I commented out the special color sections because they seemed to have some contrast issues with the new text color. I can bring them back. I'd suggest we consider switching to white text for those (at least for the brick and "graduate studies" colors).

**Comment by paregorios** [comment]
Yes, I'm sorry I wasn't clearer earlier. I didn't see any need to change the text color for the special-color sections; their contrast numbers were fine. This ticket should have been constrained only to breadcrumbs that were making use of the gray background.

**Comment by alecpm** [comment]
This is fixed on staging.

**Comment by paregorios** [comment]
Looks good. Please merge to master and deploy to production.
